ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -57,7 +57,7 @@
         raw-document (slurp (:body request))
         document (json/parse-string raw-document true)
         specification (:MetadataSpecification document)
-        spec-key (keyword (string/lower-case (:Name specification)))
+        spec-key (csk/->kebab-case-keyword (:Name specification))
```

Comment:
same comment about nil case.

Reply:
good catch, adding the default empty string to the implicit get here, for safe-guarding against nils